docs/en/siem/rules-api-create.asciidoc

```diff
@@ -15,6 +52,12 @@ A JSON object with these fields:
 |==============================================
 |Name |Type |Description |Required
 
+|actions |<<actions-object-schema, actions[]>> |Array defining the automated
+actions (notifications) taken signals are produced. |No
+
+|anomaly_threshold |Integer |Anomaly threshold above which signals are
+generated for {ml} rules |Yes, for {ml} rules.
```

Comment:
Super nitpick: should this be "...generated __by__  {ml} rules"?

In my mind, signals are generated __by__ rules, __from__ data (anomalies, events, signals), __for__ ... users? Now I'm getting semantic satiation for prepositions...

Reply:
you are 100% correct - should be `by`